src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+
+export const LoginView = ({ onLoggedIn }) => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    if (!username || !password) {
+      setError("Please enter both username and password.");
+      return;
+    }
+
+    const data = {
+      username: username,
+      password: password,
+    };
```

Comment:
Your API (https://github.com/axelgibo/movie-api/blob/f9a1f5cd63c4d23ee77cf57c4c6b88aa4b9ad183/passport.js#L13) expects this to Title case keys like this.

```suggestion
    const data = {
      Username: username,
      Password: password,
    };
```
Javscript is case-sensitive and so variables `Password` and `password` are completely different.